open_schools_platform/user_management/users/constants.py

```diff
@@ -8,4 +8,4 @@ class RegistrationConstants:
 
 class GenerateConstants:
     ALPHABET = string.ascii_letters + string.digits
-    PASSWORD_LENGTH = 10
+    PASSWORD_LENGTH = 5
```

Comment:
In open_schools_platform/user_management/users/serializers.py (51 str) writes that minimum length is equal 6.